### DISCUSSION TITLE
Does quarto support Tabbed panes for formatted code?

### DISCUSSION THREAD

**Discussion by dweng0** [quarto-dev/quarto-cli#8692] [Q&A] [answered]
### Description

I want to be able to create a tabbed code block for an example code peice in different languages. Is this something Quarto supports out of the box? I haven't seen any example in the documentation, or any extensions that explicitly mention this.

And example would be Docsys "[tabbed pane](https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes)"

**Comment by jakoblistabarth** [comment, marked as answer]
I am not sure if I get you right @dweng0 but based on your Docsys example I think [tabsets](https://quarto.org/docs/output-formats/html-basics.html#tabsets) which are build into quarto, are what you could use. Or are you looking for something else?
